metabrainz/templates/donations/donate.html

```diff
@@ -38,13 +38,17 @@
       <form id="donation-form" method="POST" class="panel-body form-horizontal" role="form">
         {{ form.hidden_tag() }}
 
-        <div id="amount-group" class="form-group">
+        <div class="form-group">
           <label for="input-amount" class="col-sm-offset-2 col-sm-3 control-label">Amount</label>
           <div class="col-sm-4">
-            <div class="input-group">
+            <div id="input-amount-fg" class="input-group">
               <span class="input-group-addon">$</span>
               {{ form.amount(id="input-amount", class="form-control", value="50.00") | safe }}</div>
           </div>
+          <div id="big-amount-info" class="col-sm-offset-3 col-sm-6" style="display:none">
+            If you want to donate specified amount, please, contact us at
```

Comment:
"If you would like to make a large donation, please contact us a donation@ before you do." 

That sounds better. :)